setup.py

```diff
@@ -47,7 +47,7 @@ def get_file(fn):
     "datadog": ["datadog"],
     "statsd": ["statsd"],
     "dev": [
-        "black==21.9b0",
+        "black==22.3.0",
```

Comment:
I haven't done a Markus release in a while, but I have fixed exactly this issue in a bunch of other libraries. Thank you for fixing this one!